linux/meshd-nl80211.c

```diff
@@ -1303,6 +1306,9 @@ meshd_parse_libconfig (struct config_setting_t *meshd_section,
     CONFIG_LOOKUP(mcast-rate, mcast_rate);
     CONFIG_LOOKUP(beacon-interval,beacon_interval);
 
+    config_setting_lookup_int(meshd_section, "protected-management-frames",
+            (config_int_t *) &config->protected_management_frames);
```

Comment:
Also should we default to "1"?  We used to assume IGTK enabled (though using MGTK), on the other hand I'm not sure if there is any kind of good backwards compatibility story here, so maybe there is no reason to choose 1 over 0.  As of kernel 4.8, we'll start encrypting the management frames instead of using BIP, so IGTK isn't necessary for HWMP frames.